summary_workflows/identification/I2_benchmark/I2_benchmark_specification.md

```diff
@@ -47,45 +59,128 @@ Fields:
 - **score** - not used, leave as "."
 - **strand** - defines the strand; either "." (=no strand) or "+" or "-".
 
-#### Format 2
+#### Input 2
 
 This BED file contains single-nucleotide position of poly(A) sites identified from the orthogonal 3'end-seq dataset.  
-Fields are the same as in Format 1.
+Fields are the same as in Input 1.
 	
+## Plots
+
+The results of this benchmark will be visualised in OpenEBench using the following plots:
+
+1. **bar plot** visualizing **Precision** of poly(A) site identification. Separate plots should be prepared for different values of distance threshold:
+
+- 10 nt
+- 50 nt
+- 100 nt
+
+**X axis** - benchmarked tool  
+**Y axis** - precision
+
+Input datasets:
+
+- RNA-Seq data compared with 3'end sequencing data
+- Simulated RNA-Seq data compared with dataset used for simulation
+
+
+Ranking: The best performing tool is the one with the highest precision value.
+
+2. **2D scatter plot** visualizing **TPR and FPR** of poly(A) site identification. Separate plots should be prepared for different values of distance threshold:
+
+- 10 nt
+- 50 nt
+- 100 nt
+
+**X axis** - FPR(d)  
+**Y axis** - TPR(d)  
+where _d - distance threshold_
+
+Input datasets:
+
+- RNA-Seq data compared with 3'end sequencing data
+- Simulated RNA-Seq data compared with dataset used for simulation
+
+Ranking: The best performing tool is the one with the highest TPR combined with lowest FPR (top left part of the plot) and the worst performing tool is the one with the lowest TPR combined with highest FPR (bottom left part of the plot. The plot should be divided into diagonal quartiles based on the distance from optimal performance. Alternatively, if the plot is divided into square quartiles, the following ranking order should be applied: top-left, top-right, bottom-left, bottom-right.
```

Comment:
This was intended as 3 separate plots (one per distance/window), so no AUC. My idea was that we cannot visualise a proper ROC in OEB, but we might still want to _look_ at how different participants perform for several representative thresholds instead of just having a single AUC number. If the plot type described in plot 4 was available, plot 2 would be unnecessary.